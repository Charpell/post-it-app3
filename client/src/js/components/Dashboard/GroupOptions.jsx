import React, { Component } from 'react'

import AppActions from '../../actions/AppActions'

/**
 * @description Component for displaying list of group options to Users
 * 
 * @class GroupOptions
 * 
 * @extends {Component}
 */
export default class GroupOptions extends Component {
    render() {
        return (
            <option>
                {this.props.keyName.groupName}
            </option>
        )
    }

}