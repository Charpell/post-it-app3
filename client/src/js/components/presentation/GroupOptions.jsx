import React from 'react';


/**
 * @description describes a stateless component
 * that renders all groups of a user
 *
 * @return { void }
 *
 * @function GroupOptions
 */
const GroupOptions = () => ({
  render() {
    return (
            <option>
                {this.props.keyName.groupName}
            </option>
    );
  }
});

export default GroupOptions;
