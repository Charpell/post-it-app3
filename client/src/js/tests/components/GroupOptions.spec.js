import React from 'react';
import { mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import GroupOptions from '../../components/Dashboard/GroupOptions'


describe('GroupOptions component', () => {
  it('It should render GroupOptions component', () => {
    const tree = renderer.create(<GroupOptions />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});

