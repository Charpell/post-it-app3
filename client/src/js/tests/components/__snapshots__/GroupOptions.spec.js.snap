// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GroupOptions component should render as expected 1`] = `<option />`;
